.github/workflows/checks.yml

```diff
@@ -0,0 +1,27 @@
+name: checks
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+env:
+  CI: true
+
+jobs:
+  checks:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16
+
+    - name: Install
+      run: yarn install --immutable --network-concurrency 1
```

Comment:
Should we add `--frozen-lockfile` here?

Reply:
i had an understanding that `--immutable` is a new `--frozen-lockfile`
<img width="797" alt="image" src="https://user-images.githubusercontent.com/1177472/212732906-5e54748a-882e-41db-b79c-6d4bc766c88f.png">